### ISSUE TITLE
Error with Output:Node Name for freezing the graph.

### ISSUE THREAD

**Issue by rahulrs2320** [mdangschat/ctc-asr#18]
Hi, I am trying to freeze the graph but when I use "bazel-bin/tensorflow/tools/graph_transforms/summarize_graph --in_graph=/path_to_file/graph.pbtxt", I get this:
How do I know which one to use amongst this?

No inputs spotted.
Found 36 variables: (name=global_step, type=int64(9), shape=[]) (name=conv/conv2d/kernel, type=float(1), shape=[11,41,1,32]) (name=conv/conv2d/bias, type=float(1), shape=[32]) (name=conv/conv2d_1/kernel, type=float(1), shape=[11,21,32,32]) (name=conv/conv2d_1/bias, type=float(1), shape=[32]) (name=conv/conv2d_2/kernel, type=float(1), shape=[11,21,32,96]) (name=conv/conv2d_2/bias, type=float(1), shape=[96]) (name=rnn/cudnn_lstm/opaque_kernel, type=float(1), shape=<unknown>) (name=dense4/dense/kernel, type=float(1), shape=[4096,2048]) (name=dense4/dense/bias, type=float(1), shape=[2048]) (name=logits/dense/kernel, type=float(1), shape=[2048,29]) (name=logits/dense/bias, type=float(1), shape=[29]) (name=beta1_power, type=float(1), shape=[]) (name=beta2_power, type=float(1), shape=[]) (name=conv/conv2d/kernel/Adam, type=float(1), shape=[11,41,1,32]) (name=conv/conv2d/kernel/Adam_1, type=float(1), shape=[11,41,1,32]) (name=conv/conv2d/bias/Adam, type=float(1), shape=[32]) (name=conv/conv2d/bias/Adam_1, type=float(1), shape=[32]) (name=conv/conv2d_1/kernel/Adam, type=float(1), shape=[11,21,32,32]) (name=conv/conv2d_1/kernel/Adam_1, type=float(1), shape=[11,21,32,32]) (name=conv/conv2d_1/bias/Adam, type=float(1), shape=[32]) (name=conv/conv2d_1/bias/Adam_1, type=float(1), shape=[32]) (name=conv/conv2d_2/kernel/Adam, type=float(1), shape=[11,21,32,96]) (name=conv/conv2d_2/kernel/Adam_1, type=float(1), shape=[11,21,32,96]) (name=conv/conv2d_2/bias/Adam, type=float(1), shape=[96]) (name=conv/conv2d_2/bias/Adam_1, type=float(1), shape=[96]) (name=rnn/cudnn_lstm/opaque_kernel/Adam, type=float(1), shape=<unknown>) (name=rnn/cudnn_lstm/opaque_kernel/Adam_1, type=float(1), shape=<unknown>) (name=dense4/dense/kernel/Adam, type=float(1), shape=[4096,2048]) (name=dense4/dense/kernel/Adam_1, type=float(1), shape=[4096,2048]) (name=dense4/dense/bias/Adam, type=float(1), shape=[2048]) (name=dense4/dense/bias/Adam_1, type=float(1), shape=[2048]) (name=logits/dense/kernel/Adam, type=float(1), shape=[2048,29]) (name=logits/dense/kernel/Adam_1, type=float(1), shape=[2048,29]) (name=logits/dense/bias/Adam, type=float(1), shape=[29]) (name=logits/dense/bias/Adam_1, type=float(1), shape=[29]) 
Found 59 possible outputs: (name=global_step/read, op=Identity) (name=global_step/cond/switch_t, op=Identity) (name=global_step/cond/switch_f, op=Identity) (name=global_step/add, op=Add) (name=seed2, op=Select) (name=IteratorToStringHandle, op=IteratorToStringHandle) (name=rnn/cudnn_lstm/Identity, op=Identity) (name=rnn/cudnn_lstm/zeros/Less, op=Less) (name=rnn/cudnn_lstm/zeros_1/Less, op=Less) (name=dense4/dense/kernel/Regularizer/l2_regularizer, op=Mul) (name=dense_to_sparse/Shape, op=Shape) (name=gradients/zeros_like, op=ZerosLike) (name=gradients/dense4/dropout/dropout/mul_grad/tuple/control_dependency_1, op=Identity) (name=gradients/dense4/dropout/dropout/truediv_grad/tuple/control_dependency_1, op=Identity) (name=gradients/dense4/Minimum_grad/tuple/control_dependency_1, op=Identity) (name=gradients/zeros_like_3, op=ZerosLike) (name=gradients/rnn/cudnn_lstm/CudnnRNN_grad/tuple/control_dependency_1, op=Identity) (name=gradients/rnn/cudnn_lstm/CudnnRNN_grad/tuple/control_dependency_2, op=Identity) (name=gradients/conv/Minimum_2_grad/tuple/control_dependency_1, op=Identity) (name=gradients/conv/Minimum_1_grad/tuple/control_dependency_1, op=Identity) (name=gradients/conv/Minimum_grad/tuple/control_dependency_1, op=Identity) (name=gradients/conv/conv2d/Conv2D_grad/tuple/control_dependency, op=Identity) (name=conv/conv2d/kernel/Adam/read, op=Identity) (name=conv/conv2d/kernel/Adam_1/read, op=Identity) (name=conv/conv2d/bias/Adam/read, op=Identity) (name=conv/conv2d/bias/Adam_1/read, op=Identity) (name=conv/conv2d_1/kernel/Adam/read, op=Identity) (name=conv/conv2d_1/kernel/Adam_1/read, op=Identity) (name=conv/conv2d_1/bias/Adam/read, op=Identity) (name=conv/conv2d_1/bias/Adam_1/read, op=Identity) (name=conv/conv2d_2/kernel/Adam/read, op=Identity) (name=conv/conv2d_2/kernel/Adam_1/read, op=Identity) (name=conv/conv2d_2/bias/Adam/read, op=Identity) (name=conv/conv2d_2/bias/Adam_1/read, op=Identity) (name=cond/switch_t, op=Identity) (name=cond/switch_f, op=Identity) (name=zeros, op=Fill) (name=rnn/cudnn_lstm/opaque_kernel/Adam/cond/switch_t, op=Identity) (name=rnn/cudnn_lstm/opaque_kernel/Adam/cond/switch_f, op=Identity) (name=rnn/cudnn_lstm/opaque_kernel/Adam/read, op=Identity) (name=cond_1/switch_t, op=Identity) (name=cond_1/switch_f, op=Identity) (name=zeros_1, op=Fill) (name=rnn/cudnn_lstm/opaque_kernel/Adam_1/cond/switch_t, op=Identity) (name=rnn/cudnn_lstm/opaque_kernel/Adam_1/cond/switch_f, op=Identity) (name=rnn/cudnn_lstm/opaque_kernel/Adam_1/read, op=Identity) (name=dense4/dense/kernel/Adam/read, op=Identity) (name=dense4/dense/kernel/Adam_1/read, op=Identity) (name=dense4/dense/bias/Adam/read, op=Identity) (name=dense4/dense/bias/Adam_1/read, op=Identity) (name=logits/dense/kernel/Adam/read, op=Identity) (name=logits/dense/kernel/Adam_1/read, op=Identity) (name=logits/dense/bias/Adam/read, op=Identity) (name=logits/dense/bias/Adam_1/read, op=Identity) (name=Adam, op=AssignAdd) (name=concat, op=ConcatV2) (name=concat_1, op=ConcatV2) (name=Merge/MergeSummary, op=MergeSummary) (name=save/Identity, op=Identity)
